app/main/routers/query/RTMediaQuery.php

```diff
@@ -676,13 +680,13 @@ function populate_media() {
 			unset( $this->media_query['lightbox'] );
 		}
 
-		// media title option
-		if ( isset( $this->media_query['media_title'] ) ) {
-			if ( 'false' === $this->media_query['media_title'] ) {
-				// Add filter show media title
+		// Media title option.
+		if ( isset( $this->media_query ) ) {
+			if ( ( isset( $this->media_query['media_title'] ) && 'false' === $this->media_query['media_title'] ) || ( true === $is_gallery_shortcode && ! isset( $this->media_query['media_title'] ) ) ) {
```

Comment:
@BhargavBhandari90 `'false' === $this->media_query['media_title']`
I think `false` should be without single quotes. 
Please check again if its boolean or not.